src/index.js

```diff
@@ -137,8 +138,12 @@ module.exports = class SentryPlugin {
     return isIncluded && !isExcluded
   }
 
+  combineRequestOptions(req) {
+    return Object.assign({}, this.requestOptions, req)
```

Comment:
I'm not sure what kind of options you're expecting, but if the config provides the `headers` hash then it will override [this](https://github.com/schneidmaster/webpack-sentry-plugin/blob/dea0c500bb6da4d3d1a9f39aa42be863dcec3f65/src/index.js#L152-L154
):

```js
      headers: {
        'Content-Type': 'application/json',
      },
```

So either do something like this:

```js
const result = Object.assign({}, this.requestOptions, req)
if (this.requestOptions.headers) {
  Object.assign(result.headers, this.requestOptions.headers, req.headers)
}
if (this.requestOptions.auth) {
  Object.assign(result.auth, this.requestOptions.auth, req.auth)
}
return result
```

Or use something like [`_.defaultsDeep`](https://lodash.com/docs/4.17.4#defaultsDeep):

```js
return defaultsDeep(req, this.requestOptions)
```

This is assuming that it's a valid use case to pass extra headers/auth options... I don't know much about Sentry instances.

Reply:
Yeah that's a good suggestion 👍 the original issue was someone wanting to configure the `strictSSL` option, so I was thinking down those lines (or maybe a case where someone needs to use `proxy` for their company network or whatever) but I think it's smart to ensure we don't clobber extra `headers`/`auth`. I can't really think of a use case but might as well account for it/future proof.

Reply:
Well if that's the case you could simply expose the `strictSSL` option... But a more general approach might cover some unknown use cases.

Reply:
Right, I figured better to allow a general passthrough to `request` options rather than implement `strictSSL` and then have future issues crop up for other use cases.